angular-portfolio/src/app/award-detail/award-detail.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Awards } from '../awards-template';
+import { AwardsService } from '../awards.service';
+
+@Component({
+  selector: 'app-award-detail',
+  templateUrl: './award-detail.component.html',
+  styleUrls: ['./award-detail.component.scss']
+})
+export class AwardDetailComponent implements OnInit {
+
+  awards: Awards[];
+  @Input() award: number;
+  hoverIndex: number;
+  current:  Awards;
+
+  constructor(private awardService: AwardsService) { }
+
+  ngOnInit(): void {
+    this.getAwards();
+  }
+
+  getAwards(): void {
+    this.awardService.getAwards()
+      .subscribe(awards => this.awards = awards);
```

Comment:
Use a block body. This means instead of "(awards => this.awards = awards)", this should be:

```
(awards => {
  this.awards = awards;
});
```

The reason for this is that the original expression body has an implicit return, meaning that the arrow function returns "awards" instead of void. It's not behavior that hurts anything in your app, but we still don't want it.